common/Geometry2d/Point.hpp

```diff
@@ -214,19 +208,19 @@ class Point {
     * rotates the point around the origin
     */
     Point& rotate(float angle) {
-        float newX = x * cos(angle) - y * sin(angle);
-        float newY = y * cos(angle) + x * sin(angle);
-        x = newX;
-        y = newY;
+        float newX = x() * cos(angle) - y() * sin(angle);
```

Comment:
Sort of inconsistent here. with between x() or _x. Either should be fine in this file though.